### DISCUSSION TITLE
There is a way to execute next build without generating static files? (just for dev enviroment)

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#23694] [Help]
Hi guys,
in my development environment, we use a pre-commit to verify if everything is ok, we use "build" command, but we have a lot of static files to generate that spend a lot of time and we just want to check if everything is building ok... There is a way to execute the Next.js build without generating static files? Just to check if everything is ok for production...

Thanks so much!

**Comment by akellbl4** [comment]
Hi, it's sad but I suppose the answer is no.
We use some verifications on commit as well, but we build the project in CI. It gives us that verification and we shouldn't wait that amount of time building the app before every commit.
